src/regfox/regfox_api.js

```diff
@@ -123,4 +125,41 @@ const getRegistrationInfoUrl = (id) => {
   return REGFOX_GET_REGISTRATION_URL.replace('${id}', id);
 };
 
-export { getRegistrationInfo, searchRegistrations, exchangeBearerToken, login, REGFOX_GRAPHQL_URL, REGFOX_EXCHANGE_TOKEN_URL, getRegistrationInfoUrl };
+/**
+ * Marks a registrant (technically a transaction of an order of a registered user) as completed (which means fully paid).
+ * See https://gist.github.com/raytingtw/8d0a083dc130c40818d71606e41b884e.
+ * Returns a promise Error if the network is down.
+ *
+ * All of these parameters can be pulled straight from the registrationInfo.
+ *
+ * @param {int} formId is the form (webpage) the user signed up with, could be hardcoded to 372652
+ * @param {int} registrationId of the registrant (**not** the Id), I think sometimes this is called the OrderId
+ * @param {int} transactionId of the registrant
+ * @param {string} id of registration (**not** the RegistrationId)
+ * @param {string} bearerToken the bearer token of the logged in user
+ */
+const markRegistrationComplete = async (formId, registrationId, transactionId, id, bearerToken) => {
+  const fullUrl = getMarkRegistrationCompleteUrl(formId, registrationId);
+  return fetch(fullUrl, {
+    method: 'PUT',
+    headers: buildHeaders(bearerToken),
+    body: JSON.stringify(buildMarkRegistrationCompleteBody(transactionId, id)),
+  }).then((response) => response.json())
+    .then((response) => {
+      if (get(response, 'code') === 1000) {
+        throw new Error(`Illegal bearerToken passed in to markRegistrationComplete: ${JSON.stringify(response)}`);
+      }
+
+      return response;
+    });
+};
+
+// Only exported for testing, DONT CALL THIS METHOD!
+const getMarkRegistrationCompleteUrl = (formId, registrationId) => {
```

Comment:
Would it be prudent to put the word `test` explicitly in this export then?

Reply:
Good call, I'll make that change!